.github/workflows/create-release-notes.yml

```diff
@@ -0,0 +1,28 @@
+name: Create Release Notes
+
+on: 
+  workflow_call:
+    inputs:
+      previous_version:
+        required: true
+        type: string
+
+jobs:
+  create-release-notes:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Create Release Notes
+        uses: actions/github-script@v7
+        env:
+          previousVersion: ${{ inputs.previous_version }}
+        with:
+          github-token: ${{ secrets.VAR_PAT }}
+          script: |
+            const path = '.github/scripts/createReleaseNotes.js';
+            const createReleaseNotes = require(path);
```

Comment:
You've done this here and elsewhere--created a variable that is only used once.  It doesn't hurt, but I would just make it one line using the literal.

Reply:
done, will see on next commit